SWLOR.Game.Server/Service/AreaTemplate.cs

```diff
@@ -0,0 +1,284 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using SWLOR.Game.Server.Core.NWNX;
+using SWLOR.Game.Server.Core.NWScript.Enum;
+using SWLOR.Game.Server.Core;
+using SWLOR.Game.Server.Entity;
+using SWLOR.Game.Server.Service.DBService;
+using SWLOR.Game.Server.Service.GuiService;
+
+namespace SWLOR.Game.Server.Service
+{
+    public class AreaTemplate
+    {
+        private static Dictionary<string, uint> TemplateAreasByResref { get; } = new();
+        private static Dictionary<uint, List<uint>> TemplateAreaCustomObjectsByArea { get; } = new();
+        private static Dictionary<int, Dictionary<int, string>> PlaceableAppearanceByPage { get; } = new();
+        private static Dictionary<int, Dictionary<int, string>> CreatureAppearanceByPage { get; } = new();
+
+        public static int TotalCountAppearancePlaceables { get; set; }
+        public static int TotalCountAppearanceCreatures { get; set; }
+
+        public const int PAGESIZE = 50;
```

Comment:
Just call this PageSize. All caps constants aren't a thing with the .NET standard.